test/ingress/aws/aws_test.go

```diff
@@ -43,8 +45,10 @@ func TestAWSIngress(t *testing.T) {
 
 	h := i.Spec.Rules[0].Host
 
+	httpsClient := getHTTPClient()
```

Comment:
Nit: I'd call this var `client` or even `c` to follow Go's recommendations for var naming in local scope.

Reply:
done.